openslides/core/static/js/core/pdf.js

```diff
@@ -4,6 +4,21 @@
 
 angular.module('OpenSlidesApp.core.pdf', [])
 
+.factory('PdfGeneralPurposeProvider', [
```

Comment:
I would prefer that this factory is not called provider because provider is a special AngularJS term used for special constructor functions.